ros_installation.md

```diff
@@ -1,46 +1,97 @@
+experience is to use the latest LTS version of ROS and the Tier 1 platform /
+operating system recommend for that version of ROS. If you host operating system
+does not match the Tier 1 operating system, consider using the Tier 1 platform
+in a virtual machine. This approach has the added benefit of not modifying your
+host OS and allowing you to roll back your mistakes.
+
+At the time of writing, our recommendation is that new users install:
+
+* [Ubuntu Jammy 22.04](https://www.releases.ubuntu.com/jammy/)
+* [ROS 2 Humble Hawksbill](https://www.ros.org/reps/rep-2000.html#humble-hawksbill-may-2022-may-2027)
+* [Gazebo Fortress](https://gazebosim.org/docs/fortress/install)
+
+# Summary of Compatible ROS and Gazebo Combinations
+
+This table includes all currently supported versions of ROS and Gazebo. All
+other ROS and Gazebo releases are end of life and we do not recommend their
+continued use.
+
+|                        	| **GZ Citadel (LTS)** 	| **GZ Fortress (LTS)** 	| **GZ Garden** 	|
+|------------------------	|----------------------	|-----------------------	|---------------	|
+| **ROS 2 Rolling**      	| X                    	| ✔️                     	| P             	|
+| **ROS 2 Humble (LTS)** 	| X                    	| ✔️                     	| P             	|
+| **ROS 2 Foxy (LTS)**   	| ✔️                    	| X                     	| X             	|
+| **ROS 1 Noetic (LTS)** 	| ✔️                    	| P                     	| X             	|
+
+
+* ✔️ - Recommended combination
+* X - Not recommended
+* P - Possible,*but not recommended*. These combinations of ROS and Gazebo can
+  be made to work together, but some effort is required.
+
+
+## Installing Gazebo with Older Versions of ROS
 
 If you are planning on using a specific version of ROS and don't have a reason
-to use a specific version of Gazebo, you should proceed with the default
-version of Gazebo available from the ROS repository when installing `ros_gz`:
+to use a specific version of Gazebo, we recommend using the default version of
+Gazebo available from the ROS repository when installing `ros_gz`. The following
+command will install the correct version of Gazebo and `ros_gz` for your ROS
+installation on a Linux system. You should replace `${ROS_DISTRO}` with your ROS
+distribution (e.g. `humble`, `rolling`,`foxy`, etc).
 
 ```bash
 sudo apt-get install ros-${ROS_DISTRO}-ros-gz
 ```
 
-This command is going to install the Gazebo libraries corresponding to the
-version detailed below in the section: "Using the default Gazebo version
-for a ROS distribution using binary installations". The rest of this
-document provides more information and other possible installations but
-at this point ROS and Gazebo are installed and ready.
+This command will install the Gazebo libraries recommended for your ROS
+installation.
 
-# Gazebo versions and ROS integration
+**The rest of this document provides more information and other, alternative,
```

Comment:
```suggestion
<div class="warning">
The rest of this document provides more information and other, alternative,
```

Reply:
The div warning displays a nice yellow block, see Note in the top of https://gazebosim.org/docs